### PR TITLE
[RPM] Update Spec file

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@dm4 @hydai PTAL.

### CODE REVIEW

File: rpm/wasmedge.spec.in [hunk 1]
```diff
@@ -45,15 +36,26 @@ cmake --build .
 %install
 cd build
 DESTDIR=%{buildroot} cmake --build . --target install
+mv %{buildroot}%{_libdir}/lib%{name}_c.so %{buildroot}%{_libdir}/lib%{name}_c.so.%{gittag}
+mv %{buildroot}%{_libdir}/%{name}/lib%{name}PluginWasmEdgeProcess.so %{buildroot}%{_libdir}/%{name}/lib%{name}PluginWasmEdgeProcess.so.%{gittag}
+ln -s lib%{name}_c.so.%{gittag} %{buildroot}%{_libdir}/lib%{name}_c.so
+ln -s lib%{name}PluginWasmEdgeProcess.so.%{gittag} %{buildroot}%{_libdir}/%{name}/lib%{name}PluginWasmEdgeProcess.so
 
 %files
+%license LICENSE LICENSE.spdx
```
Comment (lsm5) [inline rpm/wasmedge.spec.in:45]: If LICENSE.spdx is applicable to the entire package or any of the files installed by the package, the `License` line at the top of the spec file should change to: `License: ASL 2.0 and CC0`

Comment (lsm5) [reply rpm/wasmedge.spec.in:45]: If the spdx file doesn't apply to any of the files in this package, then it's ok to remove it.

Comment (dm4) [reply rpm/wasmedge.spec.in:45]: Yes, `LICENSE.spdx` applies to some files in this package. I think we should change the `License:` line to `License: ASL 2.0 and CC0`. Appreciate your help!

Comment (lsm5) [reply rpm/wasmedge.spec.in:45]: Done. Thanks

**Comment by lsm5** [comment]
@dm4 I think this PR should be good to go. But, before you merge, please check if it breaks anything in your workflow. I haven't really built the package myself yet :D . Thanks!

**Comment by lsm5** [comment]
@dm4 ohh btw, since the package name is `wasmedge` do you also want users to be able to `dnf install WasmEdge` ? I can add an additional meta-provides if that's expected behaviour.

**Comment by lsm5** [comment]
> @dm4 ohh btw, since the package name is `wasmedge` do you also want users to be able to `dnf install WasmEdge` ? I can add an additional meta-provides if that's expected behaviour.

made some more changes to account for `dnf install WasmEdge WasmEdge-devel`.

**Review by dm4 — APPROVED** [review]
Truly appreciate @lsm5's help!

**Pull request merged**